### ISSUE TITLE
Selecting disabled chapters/tables doesn't exclude them from the report

### ISSUE THREAD

**Issue by maggiewachs** [CivicActions/openacr-editor#12]
For example, if I only want to include Levels A and AA, when I produce a report I still see the Section 508 standards in the HTML, and the empty tables are included. 

Parts of the report impacted:
* "Applicable Standards/Guidelines" - only the guidelines relevant to the report should be here (currently all guidelines are there)
* Any table marked for exclusion should not be in the HTML
* Any yaml data marked for exclusion should not be in the .yaml file

**Comment by dmundra** [comment]
@maggiewachs I agree with the first two for sure.

> "Applicable Standards/Guidelines" - only the guidelines relevant to the report should be here (currently all guidelines are there)

Great point. I will fix that.

> Any table marked for exclusion should not be in the HTML

I was testing with a blank report and I am not seeing the table in the HTML when the section is disabled. Do you have a YAML example I can try?

> Any yaml data marked for exclusion should not be in the .yaml file

This one is a tricky one. The YAML essentially represents the storage of the form fields on the site and the only was I able to make the loading and downloading to work is to always have the YAML structure be consistent even though it is empty. I will try to see if we can strip that out on download and whether that loads properly again. It might but there might be blockers.

**Comment by dmundra** [comment]
A follow-up question on

> "Applicable Standards/Guidelines" - only the guidelines relevant to the report should be here (currently all guidelines are there)

We essentially set up the current feature to disable a guideline but still specify a note about it so that is why it appears in the report and in the "Applicable Standards/Guidelines" table.

If we remove items from the "Applicable Standards/Guidelines" table that are disabled but they still appear below the report, is that valid or should we add a way to completely remove a guideline from the report?

**Comment by mgifford** [comment]
I think that if a section is disabled that it should be hidden in the report and in the interface. I shouldn't see AAA values in the interface if I have excluded it. Just like I shouldn't see Hardware issues if I switch to WCAG 2.1 issues.   
  
We shouldn't delete the data if it has been entered, but it should be hidden.   
  
I think the YAML should have everything, but the HTML should be what the author has selected.   
  
I closed #19 because it's very much related and probably should be fixed at the same time.

**Comment by dmundra** [comment]
Thanks @mgifford. I can give it a go.
